backend/prisma/schema.prisma

```diff
@@ -244,69 +292,57 @@ enum EntryAwardType {
   ALLANDO_BELEPO    @map("AB")
 }
 
-model PointAwardRequest {
-  id           String         @id @default(cuid())
-  point        Int
-  awardeeId    String         @map("user_id")
-  awardee      User           @relation(fields: [awardeeId], references: [internalId])
-  guidelineId  String
-  guideline    AwardGuideline @relation(fields: [guidelineId], references: [id])
-  scoreboardId String         @map("evaluation_id")
-  scoreboard   Scoreboard     @relation(fields: [scoreboardId], references: [id])
-  discussion   Json //cellakhoz tartozo megjegyzesek
-
-  @@unique([scoreboardId, awardeeId])
+enum RequestStatus {
+  NOT_SUBMITTED
+  NOT_YET_EVALUATED
+  ACCEPTED
+  REJECTED
 }
 
-model PointAwardHistory {
```

Comment:
I couldn't figure out how this works.
I think we can just go ahead with implementing the rest of the system and implement this at the end of the MVP milestone.